### PR TITLE
Convert pywifiphisher.py to python3.

### PR DESCRIPTION
**Pull request by xcollantes**
Ran python-moderinize on one file.

### CODE REVIEW

File: wifiphisher/pywifiphisher.py [hunk 1]
```diff
@@ -346,8 +349,8 @@ def stop(self):
     def start(self):
 
         today = time.strftime("%Y-%m-%d %H:%M")
-        print ('[' + T + '*' + W + '] Starting Wifiphisher %s ( %s ) at %s' %
-               (VERSION, WEBSITE, today))
+        print(('[' + T + '*' + W + '] Starting Wifiphisher %s ( %s ) at %s' %
+               (VERSION, WEBSITE, today)))
```
Comment (cclauss) [inline wifiphisher/pywifiphisher.py]: I think you want `print()`, not `print(())`.
```python
>>> print("a", "b", "c")
a b c
>>> print(("a", "b", "c"))
('a', 'b', 'c')
```

**Comment by sophron** [comment]
Thank you @xcollantes for contributing and thanks @cclauss for all the work you've been doing on this matter.

Before we move Wifiphisher to Python3 we will have to move our modules first: 

https://github.com/wifiphisher/roguehostapd

and

https://github.com/wraith-wireless/PyRIC (which we forked here: https://github.com/wifiphisher/WiPy)

**Comment by cclauss** [comment]
Can we put some automated testing in place on these repos?  Travis CI, CircleCI, GitHub Actions, Azure Pipelines??

**Comment by xcollantes** [comment]
> LGTM. I would do **from six.moves import input, range** all on one line but different strokes for different folks.

Sounds good, I can put them both on one line for aesthetics.

**Comment by xcollantes** [comment]
@cclauss I'm not too familiar with how TravisCI works.  I tried to run unit tests but not without having to move files around to accomplish that (how the dependencies are placed).  

Will TravisCI take care of the unit tests?

**Comment by cclauss** [comment]
Look at https://travis-ci.org/wifiphisher/wifiphisher ... On that page you can see the Travis CI run that you modification just kicked off.  The Python 2 stream is clean and green and shows Travis -> tox -> pytest running the unit tests.  It [sssskips a bunch of interface tests](https://travis-ci.org/wifiphisher/wifiphisher/jobs/606643464#L736-L737) but overall tests are being run.

The Python3 tests are failing (on an issue that I believe you can fix ;-) ) but that test run is in __allow_failures__ mode for all the reasons that Travis created that mode.  https://docs.travis-ci.com/user/build-matrix/#rows-that-are-allowed-to-fail

If you want, you can install [tox](https://tox.readthedocs.io) and [pytest](https://docs.pytest.org) on your own computer so you can test while off the grid.

**Comment by sophron** [comment]
This PR looks good.

@xcollantes Can you squash your commits? This will allow us to have a cleaner history. Thanks.

**Comment by sophron** [comment]
@xcollantes If you have issues squashing your commits, feel free to close this PR and open a new one with all your changes in one commit.

**Comment by xcollantes** [comment]
Thanks for bearing with me, I don't have much practice with PR's.

**Pull request closed without merging**